Userland/Libraries/LibCrypto/Curves/SECP256r1.cpp

```diff
@@ -0,0 +1,410 @@
+/*
+ * Copyright (c) 2022, Michiel Visser <opensource@webmichiel.nl>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/ByteReader.h>
+#include <AK/Endian.h>
+#include <AK/String.h>
+#include <AK/StringBuilder.h>
+#include <AK/UFixedBigInt.h>
+#include <LibCrypto/Curves/SECP256r1.h>
+
+namespace Crypto::Curves {
+
+static u256 import_big_endian(ReadonlyBytes data)
```

Comment:
Not sure what `import` and `export` mean here. Why not `load` and `store` or `read` and `write` ?

Reply:
I might have spent too much time with the X25519 code, which uses import/export state :^)
But I agree that load or store might be nicer, I will have a look at this.

Reply:
in general, crypto code calls it import/export (see `UnsignedBigInteger::import_data`, `UnsignedBigInteger::export_data`, `RSA::import_private_key`, etc.

This is just the dominant nomenclature in sources _I've_ seen, ofc, so feel free to change the names.

Reply:
Since most of the crypto code in Serenity uses import/export right now, I guess we will keep this.